### PR TITLE
osdocs-948 updating AWS permissions in installer

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://issues.redhat.com/browse/OSDOCS-948

@jianlinliu, will you PTAL?

### CODE REVIEW

File: modules/installation-aws-permissions.adoc [hunk 1]
```diff
@@ -71,7 +71,6 @@ cluster, the IAM user requires the following permissions:
 * `ec2:CreateVpc`
 * `ec2:CreateVpcEndpoint`
 * `ec2:ModifySubnetAttribute`
-* `ec2:ModifyVpcAttribute`
```
Comment (jianlinliu) [inline modules/installation-aws-permissions.adoc:74]: Why we remove this line here? Per install source code - https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/aws/permissions.go, we still need this permission.

Comment (kalexand-rh) [reply modules/installation-aws-permissions.adoc]: Thank you @jianlinliu. I've fixed this error. Will you please take another look?

**Comment by kalexand-rh** [comment]
@jianlinliu, when you have a moment, will you please +1 this change?

**Comment by jianlinliu** [comment]
LGTM.

**Pull request merged**